src/hooks/tradingView/useTradingViewLaunchable.ts

```diff
@@ -52,13 +52,14 @@ export const useTradingViewLaunchable = ({
     if (marketId && !isDataLoading && !tvWidget) {
       const widgetOptions = getWidgetOptions(true);
       const widgetOverrides = getWidgetOverrides({ appTheme, appColorMode });
+      const languageCode = SUPPORTED_LOCALE_MAP[selectedLocale].baseTag;
```

Comment:
will these crash if the user has a (now blacklsited) locale saved in their local state?